build/shade/_k-standard-goals.shade

```diff
@@ -173,6 +173,9 @@ default SAMPLES_PROJECT_GLOB = "samples/*/project.json"
     }
   }
 
+#run-api-check .build-compile target='compile' description='Checks the compiled assemblies against a baseline for potential breaking changes'
```

Comment:
😢 Could we target the `package` target? It would be the same target `NuGetPackageVerifier` runs in.